.github/workflows/release-occm.yaml

```diff
@@ -0,0 +1,56 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Could we extend it to something like `v[0-9]+.[0-9]+.[0-9]+.*` (not sure how the syntax is!)
So we could also build things like `-alpha.X`? So you could test without needing to cleanup stuff.